packages/analytics-v1.1/src/utils/beaconQueue.js

```diff
@@ -90,10 +90,11 @@ class BeaconQueue {
   flushQueue(batch) {
     // check batch payload has data before flushing
     if (batch && batch.length > 0) {
+      const currentTime = new Date().toISOString();
```

Comment:
possible to use import { getCurrentTimeFormatted } from '@rudderstack/analytics-js-common/utilities/timestamp'; here?

Reply:
Addressed